web-console/knikubevirt/vm-details/vm-boot-order/vm-boot-order.md

```diff
@@ -0,0 +1,51 @@
+# VM details - boot order
+
+## Entering edit mode
+![VM details - Overview tab](img/B1-0-0.jpg)
+From the VM details page, Overview tab, the user clicks the pencil icon to edit the boot order.
+
+## Removing source items
+![Modal - default](img/B1-1-0.jpg)
+The modal pops up, displaying the currently listed bootable items by order of boot.
+This order was automatically set on the VM creation. This list displays bootable disks and NICS only.
+
+At this point, all the bootable sources are listed and there is no available resources to add to the list.
+An available sources counter at the bottom on the list highlights that to the user.
+
+![Removing item - actions menu](img/B1-1-1.jpg)
+From the action menu of each item, the user can remove an item from the list adding it to the available sources inventory.
+
+![Removing item](img/B1-1-2.jpg)
+An 'Add source' dropdown will appear at the bottom of the list, as the last item.
```

Comment:
The kebab action list have a delete option, and move options. The move options are another way to show the drag&drop actions (using mouse or key combinations) , if we do not display the move actions in the kebab, we will have only a delete action and will be able to use a (-) icon like in other designs, for example https://github.com/openshift/openshift-origin-design/blob/master/web-console/4.0-designs/environment/environment.md , leaving us with a more consistence experience to users, WDYT ?

Reply:
+1, I've done a bit more digging here from an accessibility standpoint and I think we can safely rely on providing better a11y support outside of the kebab so the experience is the same for all users. Take a look at this article which outlines a nice way to implement this: https://medium.com/salesforce-ux/4-major-patterns-for-accessible-drag-and-drop-1d43f64ebf09
I think with this we'll need to dig into the PatternFly side to see if this is planned for implementation or if we can possibly contribute it.
I've posted to the general channel for PatternFly to see if we can get a conversation going. With that said, I agree with @yaacov that aligning with the current pattern makes the most sense here. WDYT @Ranelim

Reply:
@yaacov we can also look at the deployments > environment for this drag and drop functionality. It aligns nicely with what we have now.